templates/shared/forms/interactive/storage.html

```diff
@@ -2,117 +2,75 @@
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_source" id="utm_source"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_content" id="utm_content"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_term" id="utm_term"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="GCLID__c" id="GCLID__c"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Facebook_Click_ID__c"
-                id="Facebook_Click_ID__c" value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" id="preferredLanguage"
-                name="preferredLanguage" maxlength="255" value="" />
-            </div>
-          </div>
-
-          <div class="u-sv2">By submitting this form, I confirm that I have read and agree to <a
+      <form action="/marketo/submit" method="post" id="mktoForm_%% formid %%" class="modal-form">
+        <div class="row">
+          <ul class="p-list col-6">
+            <li class="p-list__item">
+              <label for="firstName">First name:&ast;</label>
+              <input id="firstName" name="firstName" maxlength="255" type="text" />
+            </li>
+            <li class="p-list__item">
+              <label for="lastName">Last name:&ast;</label>
+              <input id="lastName" name="lastName" maxlength="255" type="text" />
+            </li>
+            <li class="p-list__item">
+              <label for="email">Email:&ast;</label>
+              <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$"
+                required="required" />
+            </li>
+            <li class="p-list__item">
+              <label for="phone">Phone number:</label>
+              <input id="phone" name="phone" maxlength="255" type="tel" />
+            </li>
+            <li class="p-list__item">
+              {% include "shared/forms/_country.html" %}
+            </li>
+            <li class="p-list__item">
+              <label for="company">Company Name:&ast;</label>
+              <input required id="company" name="company" maxlength="255" type="text">
+            </li>
+            <li class="p-list__item">
+              <label for="title">Job title:&ast;</label>
+              <input required id="title" name="title" maxlength="255" type="text" />
+            </li>
+            <li class="p-list__item">
+              <p>What would you like to talk to us about?</p>
+              <textarea id="about-your-project-or-interest" name="sector-specific-requirements"
+              aria-label="What would you like to talk to us about?" rows="3"></textarea>
```

Comment:
Better to use semantic html (label instead of p tag) and to keep the original `Comments_from_lead__c` id/name/etc. The fields you pass in the request have to match what marketo is expecting or the request will fail